tests/e2e/cucumber/no-results.feature

```diff
@@ -0,0 +1,11 @@
+Feature: No-results component
+
+  Scenario Outline: 1. No results message exists and contains the query
```

Comment:
the `contains the query` is not really mandatory. It is the usual case, but may be some customer with a message without the query